gum/backend-arm/gumstalker-arm.c

```diff
@@ -429,6 +432,15 @@ static void gum_exec_block_thumb_close_prolog (GumExecBlock * block,
 static gboolean gum_stalker_is_thumb (gconstpointer address);
 static gboolean gum_stalker_is_kuser_helper (gconstpointer address);
 
+static void gum_generator_context_advance_exclusive_load_offset (
+    GumGeneratorContext * gc);
+static gboolean gum_generator_context_is_timing_sensitive (
+    GumGeneratorContext * gc);
```

Comment:
These should be before the group above, as they don't belong to any class. (They're free-standing helpers.)